### ISSUE TITLE
Not all plugins are shown in GIMP after unsuccessful installation

### ISSUE THREAD

**Issue by datalot-369** [kritiksoman/GIMP-ML#19]
**Describe the bug**

I have installed GIMP-ML as the main README.MD file describes. After all, JUST these plugins are shown:

`Layer > GIML-ML >`
`deeplabv3...`
`facegen...`
`faceparse...`
`kmeans...`

> I can't infer exactly how many plugins this package provide because de Manual, my menu, and the YouTube channel are not homogeneous.

**Steps to reproduce**

1. Follow the [README.MD file installations steps](https://github.com/kritiksoman/GIMP-ML/blob/master/README.md#installation-steps).
2. I did receive [these terminal output errors](https://github.com/kritiksoman/GIMP-ML/files/5097609/out.txt).
3. Open GIMP.
4. Expand the `Layer > GIML-ML` menu.

**Screenshots**

![image](https://user-images.githubusercontent.com/21994415/90658715-e6a33b00-e211-11ea-8285-ecdc45906137.png)

**Desktop:**

```
 - OS Version: MX Linux 19.2
 - GIMP Version: 20.10.20
```

**Comment by YAFU** [comment]
Hi. I have been able to successfully install the plugin on Kubuntu 18.04 by following new installation instructions, doing the process completely from scratch.
Still no luck on Kubuntu 20.04.

**Comment by datalot-369** [comment]
Well, I will try them and say you all if I do it successfully.
I hesitate about the `[Work in progress.] Simplified installation and updating method.` line.
Honestly, I prefer an ArchOS-like documentation instead of generic instructions,
That should have a conveniently separated INSTALLATION.MD file referenced by the main README.MD to explain the process and its issues' workarounds in-deep.

**Comment by kritiksoman** [comment]
it is fixed. will make better documentation.

**Comment by datalot-369** [comment]
I have written a great comment yesterday but in a system restart GitHub losses all. (-_-)'

TL;DR: After deleting all related to the first try and re-doing a fresh git-clone to follow the new instructions, all is done properly and now I have the following entries (respectively):

- `colorpalette...`
- `deepblur...`
- `deep color...`
- `deeplabv3...`
- `deepmatting...`
- `facegen...`
- `faceparse...`
- `kmeans...`
- `MonoDepth...`
- `super-resolution...`
- `update...`

![image](https://user-images.githubusercontent.com/21994415/91224493-32f9e980-e6f0-11ea-9cb1-63cd240a861c.png)

I would suggest adding to the manual a description with several before-after examples, recommendations, and requirements to run every "filter".

Also, I have some questions:
1. When should I click `update...` again?
2. What is the `weights` source?
3. Where can I get involved with the specific weights behind? (I would like to know news about the unblur, e.g.)

I think these questions could be part of a FAQ.
